### PR TITLE
Add section on community builds.

### PR DESCRIPTION
**Pull request by ChristopherSchultz**
Also add headings.

### CODE REVIEW

File: achievements/tomcat.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 Prior to the Hackathon the Tomcat commiters created a [wiki page](https://cwiki.apache.org/confluence/display/TOMCAT/EU+FOSSA+May+2019) with topics to work on and discuss.
 
-Most of the people at the Tomcat table got interested in getting Tomcat to run on GraalVM. 
+# GraalVM
```
Comment (jfhovinne) [inline achievements/tomcat.md]: Should be a h2 then ?

File: achievements/tomcat.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+Some of the people at the Tomcat table got interested in getting Tomcat to run on GraalVM. 
 Because we already knew that Tomcat would not run with the latest published version of GraalVM, we started with compiling the master branch of GraalVM from Github. There is a commit in master just done a few days before the event that should fix some issues that appeared before.
 
-Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
\ No newline at end of file
+Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
+
+## Community and Builds
+The team instructed a few attendees on how to build Apache Tomcat from source either from q git-checkout or from a release package. Also gave instructions for forking on GitHub for future hacking and pull-requests.
```
Comment (jfhovinne) [inline achievements/tomcat.md]: "either from checking out the code" maybe ?
"Also gave instructions regarding GitHub forks for future ..."

**Review by jfhovinne — CHANGES_REQUESTED** [review]
Thanks @ChristopherSchultz 
See comments

File: achievements/tomcat.md (hunk 1 above)
Comment (ChristopherSchultz) [reply achievements/tomcat.md]: Yes. Fixed.

File: achievements/tomcat.md (hunk 2 above)
Comment (ChristopherSchultz) [reply achievements/tomcat.md]: I hope it's clearer, now.

File: achievements/keysigning.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# PGP Key Signing
+
+Several attendees expressed interested in participating in the key signing along with the Apache committers.
```
Comment (jfhovinne) [inline achievements/keysigning.md]: "expressed interest"

File: achievements/tomcat.md [hunk 4]
```diff
@@ -2,7 +2,17 @@
...
+## GraalVM
+Some of the people at the Tomcat table got interested in getting Tomcat to run on GraalVM. 
 Because we already knew that Tomcat would not run with the latest published version of GraalVM, we started with compiling the master branch of GraalVM from Github. There is a commit in master just done a few days before the event that should fix some issues that appeared before.
 
-Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
\ No newline at end of file
+Compiling GraalVM from source turned out to be a little bit tricky, because it did not compile with a regular version of the Java VM but required a special JVM.
+
+## Community and Builds
+The team instructed a few attendees on how to build Apache Tomcat from source either from checkout-oput the code from git or from a release package. Also gave instructions for forking the project on GitHub for future hacking and pull-requests.
```
Comment (jfhovinne) [inline achievements/tomcat.md]: "checkout-oput" ?

File: achievements/tomcat.md [hunk 5]
```diff
@@ -2,7 +2,17 @@
...
+
+## Fixes
+
+One enhancement request was completed fixed.
+
+One new bug was filed.
```
Comment (jfhovinne) [inline achievements/tomcat.md:18]: "completed fixed" ?
Could you use a list ? Thanks.

**Review by jfhovinne — CHANGES_REQUESTED** [review]

File: achievements/keysigning.md (hunk 3 above)
Comment (ChristopherSchultz) [reply achievements/keysigning.md]: Fixed.

File: achievements/tomcat.md (hunk 4 above)
Comment (ChristopherSchultz) [reply achievements/tomcat.md]: Fixed.

File: achievements/tomcat.md (hunk 5 above)
Comment (ChristopherSchultz) [reply achievements/tomcat.md:18]: Yes, "completed fixed", as opposed to "completed rejected" or some other kind of resolution. Feel free to change formatting. This page is for *your* group's achievements, not the Tomcat project's. We already have separate tracking mechanisms, etc.

**Comment by jfhovinne** [comment]
Superseded by other PRs.

**Pull request closed without merging**